lua/nvim-treesitter.lua

```diff
@@ -1,13 +1,20 @@
 local api = vim.api
 local parsers = require'nvim-treesitter.parsers'
 local install = require'nvim-treesitter.install'
+local locals = require'nvim-treesitter.locals'
 
 local M = {}
 
 -- This function sets up everythin needed for a given language
 -- this is the main interface through the plugin
 function M.setup(lang)
   if parsers.has_parser(lang) then
+    if locals.is_supported(lang) then
+      print("Locals setup for", lang)
```

Comment:
space after for

Reply:
Actually, the space is directly inserted by `print`.

Reply:
my bad, i usually do "str " .. var, that's why i got confused